### PR TITLE
Add ansible auto-publisher image

### PR DESCRIPTION
**Pull request by SuperQ**
Add a Docker image to simplify publishing of new releases.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

**Comment by SuperQ** [comment]
Ping :)

File: .circleci/config.yml [hunk 1]
```diff
@@ -66,6 +91,17 @@ workflows:
...
+        filters:
+          branches:
+            ignore: master
+    - publish-image:
+        context: quay.io
+        requires:
+        - build-image
+        filters:
+          branches:
+            only: master
```
Comment (roidelapluie) [inline .circleci/config.yml:104]: if we require build-images but build-images ignores the branch master will it work?

Comment (SuperQ) [reply .circleci/config.yml:104]: You're right, we have the same mistake in golang-builder.

**Comment by SuperQ** [comment]
Ping again.

**Review by paulfantom — APPROVED** [review]
lgtm

**Pull request merged**